utilities/AFQ_mrtrix_track.m

```diff
@@ -0,0 +1,168 @@
+function [status, results, fg, pathstr] = AFQ_mrtrix_track(files, ...
+                                                           roi, ...
+                                                           mask, ...
+                                                           mode, ...
+                                                           nSeeds, ...
+                                                           bkgrnd, ...
+                                                           verbose, ...
+                                                           clobber, ...
+                                                           mrtrixVersion)
+%
+% function [status, results, fg, pathstr] = mrtrix_track(files, roi, mask, mode, nSeeds, bkgrnd, verbose)
+%
+% Provided a csd estimate, generate estimates of the fibers starting in roi 
+% and terminating when they reach the boundary of mask
+%
+% Parameters
+% ----------
+% files: structure, containing the filenames generated using mrtrix_init.m
+% roi: string, filename for a .mif format file containing the ROI in which
+%      to place the seeds. Use the *_wm.mif file for Whole-Brain
+%      tractography.
+% mask: string, filename for a .mif format of a mask. Use the *_wm.mif file for Whole-Brain
+%      tractography.
+% mode: Tracking mode: {'prob' | 'stream'} for probabilistic or
+%       deterministic tracking. 
+% nSeeds: The number of fibers to generate.
+% bkgrnd: on unix, whether to perform the operation in another process
+% verbose: whether to display standard output to the command window. 
+% clobber: Whether or not to overwrite the fiber group if it was already
+%          computed
+% 
+% Franco, Bob & Ariel (c) Vistalab Stanford University 2013 
+% Edit GLU 06.2016 added mrTrix versioning
+
+status = 0; results = [];
+if notDefined('verbose'),  verbose = false;end
+if notDefined('bkgrnd'),    bkgrnd = false;end
+if notDefined('clobber'),  clobber = false;end
+
+% Choose the tracking mode (probabilistic or stream)
+% mrTrix3 new options
+% -algorithm name
+%      specify the tractography algorithm to use. Valid choices are: FACT, iFOD1,
+%      iFOD2, Nulldist1, Nulldist2, SD_Stream, Seedtest, Tensor_Det, Tensor_Prob
+%      (default: iFOD2).
+switch mode
+  case {'prob','probabilistic tractography'}
+    mode_str2 = 'SD_PROB';
+    mode_str3 = 'Tensor_Prob';
```

Comment:
This should be SD_Prob I think. We don't want to track based on the tensor. We want to track based on SD